XamlControlsGallery/ControlPages/ListViewPage.xaml

```diff
@@ -72,29 +72,91 @@
             </Grid>
         </DataTemplate>
 
-        <DataTemplate x:Key="ContactListViewTemplate2" x:DataType="local1:Contact">
-            <Grid Margin="0,12,0,12">
-                <Grid.ColumnDefinitions>
-                    <ColumnDefinition Width="Auto" MinWidth="550"/>
-                </Grid.ColumnDefinitions>
-                <TextBlock Grid.Column="1"
-                           Text="{x:Bind Name}"
-                           x:Phase="1"
-                           Margin="0,12,0,0"/>
+        <DataTemplate x:Key="MessageViewTemplate" x:DataType="local1:Message">
+            <Grid>
+                <Grid.RowDefinitions>
+                    <RowDefinition Height="Auto"/>
+                </Grid.RowDefinitions>
+                <Grid Height="Auto" Grid.Row="1" Margin="5" HorizontalAlignment="{x:Bind MsgAlignment}">
+                    <Grid.ColumnDefinitions>
+                        <ColumnDefinition Width="*"/>
+                    </Grid.ColumnDefinitions>
+                    <StackPanel Height="75" Width="350" Padding="10 0 0 10" Background="{x:Bind BgColor}" CornerRadius="4">
+                        <TextBlock Text="{x:Bind MsgText}" Padding="0 10 0 0" FontSize="20" Foreground="White"></TextBlock>
+                        <TextBlock Text="{x:Bind MsgDateTime}" Padding="0 0 0 10" FontSize="15" Foreground="White"></TextBlock>
+                    </StackPanel>
+                </Grid>
             </Grid>
         </DataTemplate>
+
+        <DataTemplate x:Key="ContactListViewTemplate2" x:DataType="local1:Contact">
+            <TextBlock Text="{x:Bind Name}" x:Phase="1" Margin="0,5,0,5"/>
+        </DataTemplate>
+        
     </Page.Resources>
     <StackPanel>
-        <local:ControlExample HeaderText="A Simple ListView with Selection support">
-            <ListView
+        
+        <local:ControlExample HeaderText="Basic ListView with Simple DataTemplate">
+            <StackPanel>
+                <TextBlock Margin="0 0 0 15">
+                    This is a basic ListView that has the full source code below. <LineBreak/>Other samples on this page display only the additional markup needed to customize a simple ListView like this one.
+                </TextBlock>
+                <ListView
+                x:Name="BaseExample"
```

Comment:
I prefer the convention that puts the x:Name on the same line as the control declaration (`<ListView` in this case). Makes the code a little less tall :)